modules/ppcp-onboarding/src/State.php

```diff
@@ -109,37 +115,37 @@ public function production_state() : int {
 		);
 	}
 
+	/**
+	 * Translates an onboarding state to a string.
+	 *
+	 * @param int $state An onboarding state to translate.
+	 * @return string A string representing the state: "start" or "onboarded".
+	 */
+	public static function get_state_name( int $state ) : string {
+		switch ( $state ) {
+			case self::STATE_START:
+				return 'start';
+			case self::STATE_ONBOARDED:
+				return 'onboarded';
+			default:
+				return 'unknown';
+		}
+	}
+
 	/**
 	 * Returns the state based on progressive and onboarded values being looked up in the settings.
```

Comment:
Maybe update description (... _based on progressive_ ...) here as progressive is not used anymore.